vllm/model_executor/models/mixtral.py

```diff
@@ -66,13 +68,15 @@ def __init__(
         intermediate_size: int,
         params_dtype: Optional[torch.dtype] = None,
         tp_size: Optional[int] = None,
+        use_fp8: bool = True,
```

Comment:
`mixtral_quant.py` is currently a bit of a hack for the quantized models that don't support the fused_moe kernel. Since with fp8 we support the fused_moe kernel, we can use `mixtral.py` -- hopefully in the future we can get rid of `mixtral_quant.py`. I agree with your suggestion to do something like `quantize_method`, will try that out :)